tests/admin_filters/models.py

```diff
@@ -10,6 +10,24 @@
 
 
 @python_2_unicode_compatible
+class Department(models.Model):
```

Comment:
It would make the diff less confusing if you didn't reorder the models. You can use `ForeignKey('Employee')` if that's the main reason for the move.